README.md

```diff
@@ -39,7 +39,7 @@ This should work straight-forward. Install a rust nightly, clone the repo, and t
 ## Building and running locally on OS X
 
 ``` bash
-$ brew install openssl ... ... ... # TODO: other build dependencies?
+$ brew install openssl
```

Comment:
Fabrice, should have written that, yes.. I removed brew and installed just openssl and multirust again and it worked.